website/docs/d/storage_containers.html.markdown

```diff
@@ -0,0 +1,57 @@
+---
+subcategory: "Storage"
+layout: "azurerm"
+page_title: "Azure Resource Manager: Data Source: azurerm_storage_containers"
+description: |-
+  Gets information about an existing Storage Containers.
+---
+
+# Data Source: azurerm_storage_containers
+
+Use this data source to access information about an existing Storage Containers.
+
+## Example Usage
+
+```hcl
+data "azurerm_storage_containers" "example" {
+  storage_account_id = "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg1/providers/Microsoft.Storage/storageAccounts/sa1"
+}
+
+output "container_id" {
+  value = data.azurerm_storage_containers.example.containers.0.resource_manager_id
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `storage_account_id` - (Required) The ID of the Storage Account that the Storage Containers reside in.
+
+---
+
+* `name_prefix` - (Optional) A prefix match used for the Storage Container `name` field, case sensitive.
```

Comment:
Terraform is case-sensitive by default (like most systems are), so I don't think we need to call this one out explicitly?

```suggestion
* `name_prefix` - (Optional) A prefix match used for the Storage Container `name` field.
```